<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurveyDesignDialogBase</class>
 <widget class="QDialog" name="SurveyDesignDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Survey design</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="mAddBaselineLabel">
       <property name="text">
        <string>Add baseline</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="mAddBaselineToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/plugins/survey_design/mActionToggleEditing.png</normaloff>:/plugins/survey_design/mActionToggleEditing.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mAddStratumLabel">
       <property name="text">
        <string>Add stratum</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="mAddStratumToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/plugins/survey_design/mActionToggleEditing.png</normaloff>:/plugins/survey_design/mActionToggleEditing.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="mAddSurveyAreaLabel">
       <property name="text">
        <string>Add survey area</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="mAddSurveyAreaToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/plugins/survey_design/mActionToggleEditing.png</normaloff>:/plugins/survey_design/mActionToggleEditing.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="mPointSurveyRadioButton">
       <property name="text">
        <string>Point survey</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mTransectSurveyRadioButton">
       <property name="text">
        <string>Transect survey</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="mShareBaselineCheckBox">
       <property name="text">
        <string>Share baseline</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>196</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="mCreateSampleButton">
     <property name="text">
      <string>Create sample</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SurveyDesignDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SurveyDesignDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
